src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse41.PlatformNotSupported.cs

```diff
@@ -14,387 +14,459 @@ namespace System.Runtime.Intrinsics.X86
 
         /// <summary>
-        /// __m128i _mm_blend_epi16 (__m128i a, __m128i b, const int imm8)
+        /// __m128i _mm_blend_epi16 (__m128i a, __m128i b, const int imm8); PBLENDW xmm, xmm/m128 imm8
         /// </summary>
         public static Vector128<ushort> Blend(Vector128<ushort> left, Vector128<ushort> right, byte control) { throw new PlatformNotSupportedException(); }
 
         /// <summary>
-        /// __m128 _mm_blend_ps (__m128 a, __m128 b, const int imm8)
+        /// __m128 _mm_blend_ps (__m128 a, __m128 b, const int imm8); BLENDPS xmm, xmm/m128, imm8
         /// </summary>
         public static Vector128<float> Blend(Vector128<float> left, Vector128<float> right, byte control) { throw new PlatformNotSupportedException(); }
 
         /// <summary>
-        /// __m128d _mm_blend_pd (__m128d a, __m128d b, const int imm8)
+        /// __m128d _mm_blend_pd (__m128d a, __m128d b, const int imm8); BLENDPD xmm, xmm/m128, imm8
         /// </summary>
         public static Vector128<double> Blend(Vector128<double> left, Vector128<double> right, byte control) { throw new PlatformNotSupportedException(); }
-        
+
         /// <summary>
-        /// __m128i _mm_blendv_epi8 (__m128i a, __m128i b, __m128i mask)
+        /// __m128i _mm_blendv_epi8 (__m128i a, __m128i b, __m128i mask); PBLENDVB xmm, xmm/m128, xmm
         /// </summary>
         public static Vector128<sbyte> BlendVariable(Vector128<sbyte> left, Vector128<sbyte> right, Vector128<sbyte> mask) { throw new PlatformNotSupportedException(); }
         /// <summary>
-        /// __m128i _mm_blendv_epi8 (__m128i a, __m128i b, __m128i mask)
+        /// __m128i _mm_blendv_epi8 (__m128i a, __m128i b, __m128i mask); PBLENDVB xmm, xmm/m128, xmm
         /// </summary>
         public static Vector128<byte> BlendVariable(Vector128<byte> left, Vector128<byte> right, Vector128<byte> mask) { throw new PlatformNotSupportedException(); }
         /// <summary>
-        /// __m128 _mm_blendv_ps (__m128 a, __m128 b, __m128 mask)
+        /// __m128 _mm_blendv_ps (__m128 a, __m128 b, __m128 mask); BLENDVPS xmm, xmm/m128, xmm0
         /// </summary>
         public static Vector128<float> BlendVariable(Vector128<float> left, Vector128<float> right, Vector128<float> mask) { throw new PlatformNotSupportedException(); }
         /// <summary>
-        /// __m128d _mm_blendv_pd (__m128d a, __m128d b, __m128d mask)
+        /// __m128d _mm_blendv_pd (__m128d a, __m128d b, __m128d mask); BLENDVPD xmm, xmm/m128, xmm0
         /// </summary>
         public static Vector128<double> BlendVariable(Vector128<double> left, Vector128<double> right, Vector128<double> mask) { throw new PlatformNotSupportedException(); }
 
         /// <summary>
-        /// __m128 _mm_ceil_ps (__m128 a)
+        /// __m128 _mm_ceil_ps (__m128 a); ROUNDPS xmm, xmm/m128, imm8(10)
```

Comment:
Would it be clearer to write the implicit immediate as `imm8=0b10`?

Reply:
Probably. 👍